Dockerfile

```diff
@@ -20,6 +20,7 @@ WORKDIR $WORKDIR
 
 COPY . $WORKDIR
 COPY docker-assets/entrypoint /usr/bin
+COPY docker-assets/run_persister /usr/bin
```

Comment:
I'm wondering if it makes more sense to name this less specifically, something like `run_the_thing` (:laughing:)?  Makes copy-pasting to different dockerfiles easier, maybe?

Reply:
Or just `run`?

Reply:
Eh, I'd rather have it be a bit descriptive. I guess I'd also rather not have people copying and pasting to create new images. This is the one thing that someone would have to change so we can indicate that by having a different name.